gadget/install/partition.go

```diff
@@ -90,12 +90,12 @@ func createMissingPartitions(dl *gadget.OnDiskVolume, pv *gadget.LaidOutVolume)
 // returns a partitioning description suitable for sfdisk input and a
 // list of the partitions to be created.
 func buildPartitionList(dl *gadget.OnDiskVolume, pv *gadget.LaidOutVolume) (sfdiskInput *bytes.Buffer, toBeCreated []gadget.OnDiskStructure) {
-	sectorSize := dl.SectorSize
+	sectorSize := uint64(dl.SectorSize)
 
 	// Keep track what partitions we already have on disk
-	seen := map[quantity.Offset]bool{}
+	seen := map[uint64]bool{}
```

Comment:
When this was `quantity.Offset` it somehow helped to explain how this map works, because it made clear that the keys are offsets. So if you do this change, it would be nice to update the comment above and add a line like "the map uses the starting sector numbers of the partitions as keys"

Reply:
sure I can update the comment about this

Reply:
done